### DISCUSSION TITLE
save Links not working (urgent)

### DISCUSSION THREAD

**Discussion by getch04** [isar/isar#1466] [Help]
i have 2 links and when i try to save it is not working , nay help is appreciable 

or is there any way to change the following links to embedded objects:
![image](https://github.com/isar/isar/assets/39654442/aff119f3-28c1-40c7-a516-f6f4c8042c33)

![image](https://github.com/isar/isar/assets/39654442/b41a81de-7538-4570-88ff-59a351b286d8)

**Comment by JsonLinesCode** [comment]
Hey, make sure you've previously added the linked objects to the database.
